### PR TITLE
Bug 1764027: test: add more debugging output to oauth-server test

### PR DESCRIPTION
**Pull request by mfojtik**

### CODE REVIEW

**Comment by stlaz** [comment]
/hold
I shouldn’t review from phone, there might be a bug here

File: test/extended/util/oauthserver/oauthserver.go [hunk 1]
```diff
@@ -244,6 +246,19 @@ func waitForOAuthServerRouteReady(oc *exutil.CLI) error {
 		}
 		return false, nil
 	})
+	if waitErr != nil {
+		serverPod, err := oc.AdminKubeClient().CoreV1().Pods(oc.Namespace()).Get("test-oauth-server", metav1.GetOptions{})
+		if err != nil {
+			e2e.Logf("unable to get %q pod: %v", "test-oauth-server", err)
+		}
+		e2e.Logf("%q pod status: %s", "test-oauth-server", spew.Sdump(serverPod.Status))
```
Comment (stlaz) [inline test/extended/util/oauthserver/oauthserver.go]: nil deref here (serverPod)

Comment (mfojtik) [reply test/extended/util/oauthserver/oauthserver.go]: fixed

**Pull request closed without merging**